src/plugins/logchange/README.md

```diff
@@ -1,5 +1,6 @@
 - infos = Information about the logchange plugin is in keys below
 - infos/author = Markus Raab <elektra@libelektra.org>
+- infos/maintainer = Florian Lindner <florian.lindner@student.tuwien.ac.at>
```

Comment:
any particular reason why you take maintainership of logchange? That would be more suited for me

Reply:
Good point, you're absolutely right!
I removed me as a maintainer for `logchange` and instead chose the `length`-plugin.